query/repo/repo_webhook_package_registery_security_settings_enabled.sql

```diff
@@ -1,6 +1,6 @@
 with hooks_info as (
   select
-    name,
+    login as name,
```

Comment:
```suggestion
    login,
```

Can we use the new column name and then update the rest of the query below?

Reply:
This is used to align column names in a `union`, I swapped from using `name` to `login` as this is what's used for org related output in most places. Can revert to using `name` if preferred?